src/crimson/os/seastore/seastore.cc

```diff
@@ -1307,8 +1309,10 @@ SeaStore::tm_ret SeaStore::_omap_set_header(
 {
   LOG_PREFIX(SeaStore::_omap_set_header);
   DEBUGT("{} {} bytes", *ctx.transaction, *onode, header.length());
-  assert(0 == "not supported yet");
-  return tm_iertr::now();
+  std::string name("omap_header");
```

Comment:
Define a constant for "omap_header" with a brief comment explaining that we are using a special xattr value to store the omap header.

Reply:
done.